app/models/concerns/metadata.rb

```diff
@@ -40,7 +40,9 @@ module Metadata
     end
 
     # For Merritt ARKs
-    property :merritt_id, predicate: RDF::Vocab::DC11.identifier
+    property :merritt_id, predicate: RDF::Vocab::DC11.identifier do |index|
```

Comment:
Is the plan to populate this field for all objects (including newly ingested objects without a locally defined ARK?). If so :+1:, if not, we may have a gap to close the JIRA ticket.

Reply:
This property is now limited to ETD objects only as defined [here](https://github.com/ucsblibrary/alexandria/pull/183/files#diff-deca340d3f65dbe0cf9c3b44fb37f018R37)